ecs-agent/daemonimages/csidriver/util/fs/fs.go

```diff
@@ -0,0 +1,51 @@
+//go:build linux || darwin
+// +build linux darwin
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package fs
+
+import (
+	"golang.org/x/sys/unix"
+)
+
+type UsageInfo struct {
+	Bytes  int64
+	Inodes int64
+}
+
+// Info linux returns (available bytes, byte capacity, byte usage, total inodes, inodes free, inode usage, error)
+// for the filesystem that path resides upon.
+func Info(path string) (int64, int64, int64, int64, int64, int64, error) {
```

Comment:
Who will be consuming this function? Is it possible to return the information in a struct? That's way too many unnamed `int64`s.

Reply:
How are we testing this function?

Reply:
This isn't used for this pR but will be used in a follow up PR.

For the testing, this will invoke a syscall. No idea for now about how to test this simple file.

For the naming, let me use named return parameter to give callers a better understanding for each var.

Reply:
Okay. Have we tested this function manually for both Linux and Windows?

Reply:
Just for linux. Window team will work on this later I think.

Reply:
I see. Let's remove this code from this PR and add it back when it's tested. The code is non-trivial so we should test it when checking it in.